backend/app/services/summary_service.py

```
@@ -0,0 +1,519 @@
+            summary_data = {
+                "status": "failed",
+                "error": str(e),
+                "updated_at": datetime.now().isoformat(),
+            }
+            document.summary = summary_data
+            flag_modified(document, "summary")
+            self.db.commit()
+            return None
+
+    async def get_document_summary(
+        self, document_id: int, user_id: int
```

Ruff (0.14.10)

Unused method argument: `user_id`

(ARG002)